packages/react-magma-dom/src/i18n/default.ts

```diff
@@ -144,8 +144,9 @@ export const defaultI18n: I18nInterface = {
         displayValue: 'ESC',
         explanation: 'Return to the date input field.',
       },
-      questionMark: {
```

Comment:
changing this would be a breaking change for our adopters if they are already using `questionMark`, so maybe we should leave it as `questionMark` even if it's not 100% accurate

Reply:
Oh, okay. I thought I'd better clarify that and change from `questionMark` to `ctrlShiftAndQuestionMark` everywhere in the code. Changed it back to `questionMark` :)